app/views/step_by_step_pages/show/_actions.erb

```diff
@@ -3,12 +3,11 @@
     <% if @step_by_step_page.can_be_published? || @step_by_step_page.can_discard_changes? || @step_by_step_page.scheduled_for_publishing? || !@step_by_step_page.steps_have_content? %>
       <% if @step_by_step_page.scheduled_for_publishing? %>
         <p class="govuk-body">Scheduled to be published at <%= format_full_date_and_time(@step_by_step_page.scheduled_at) %></p>
-        <%= form_tag step_by_step_page_unschedule_path(@step_by_step_page), method: :unschedule do %>
-          <%= render "govuk_publishing_components/components/button", {
-            text: "Unschedule",
-            destructive: true
-          } %>
-        <% end %>
+        <%= link_to "Unschedule",
+          step_by_step_page_unschedule_path(@step_by_step_page),
+          method: :post,
```

Comment:
Can you update the commit message? It's still mentions changing the route to a GET.

Reply:
Updated.